src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -751,6 +751,12 @@ public static class MsalError
         /// </summary>
         public const string BrokerApplicationRequired = "broker_application_required";
 
+        /// <summary>
+        /// <para>What happens?</para>You have configured a claims request, but the claims string is not in JSON format
+        /// <para>Mitigation</para>Ensure that the claims parameter is valid JSON. Inspect the inner exception for details about JSON parsing. 
+        /// </summary>
+        public const string ClaimsNotJson = "claims_not_json";
```

Comment:
nit: Maybe ClaimsNotValidJson - it's a little confusing what is meant here

Reply:
+1. ClaimsAreInvalidJson, InvalidJsonClaimsFormat.